index.html

```diff
@@ -26,5 +26,17 @@ <h1>Seytech</h1>
         </div>
         <div class="col-md-1"></div>
     </header>
+    <!-- here start the slider -->
+    <div class="row slider-unique">
+        <div class="col-md-3"></div>
+        <div class="col-md-6">
+            <h2>Build a landing page for your business or project and generate more leads!</h2>
+            <div>
+                <input placeholder="enter your email"/>
```

Comment:
hey man your code is very bad

Reply:
umm i think we need to follow this approach because of something

Reply:
pretty nice:)